src/omnisharp/server.ts

```diff
@@ -259,6 +260,23 @@ export class OmniSharpServer {
             args.push('--debug');
         }
 
+        let experimentalVersion : string;
+        let experimentalOption = this._options.experimentalOmnisharp;
+        let extensionPath = utils.getExtensionPath();
+        
+        if(experimentalOption == "latest"){
+            let downloader = new OmnisharpDownloader();
```

Comment:
Is this expected to actually download something from the Internet somehow? If so, should this be plugged into the package downloader that already exists in omnisharp-vscode?